javascript/index.js

```diff
@@ -13,16 +13,27 @@ let milDec = document.getElementById('milDec');
 let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
-function printTime() {
-  // ... your code goes here
+function printTime(minutes, seconds) {
+  printMinutes(minutes)
+  printSeconds(seconds)
```

Comment:
same here, printTime function should not have any parameters, same for the printMinutes and printSeconds